www/taggame

```diff
@@ -62,16 +62,12 @@ if ($db == false)
 $xml = isset($_REQUEST['xml']);
 if (!$xml) {
     if ($_SERVER['REQUEST_METHOD'] != 'POST') {
-        http_response_code(405);
-        echo "405 Method Not Allowed";
-        exit();
+        sendResponse("405 Method Not Allowed", "Not Saved", "This API requires you to use HTTP method POST");
```

Comment:
http_response_code officially only accepts integers. We should not be passing strings (rest of the code should also be fixed.

The "not saved" label is just for our UI and probably not relevant in this case, but can be left in.

Reply:
I'm not sure I understand this comment. I think you might be suggesting that I should change this line to say `sendResponse(405, "Not Saved", "blah blah")`?

But `sendResponse` in `taggame` doesn't use `http_response_code`. It uses `header("HTTP/1.1 $status");`, which is where we _do_ have to include the status text ourselves.

I guess we could have a separate PR to replace instances of `header("HTTP/1.1 $status");` but it's probably not worth the effort.

Reply:
You're right, I had the two approaches mixed up!